source/containers/Form/hooks/formActions.ts

```diff
@@ -317,6 +318,19 @@ export function validateAnswer(
   return state;
 }
 
+function shouldValidateAnswer(validationItem, formAnswers, allQuestions) {
+  if(validationItem) {
+    if(validationItem.hasCondition && validationItem.conditionalOn) {
+      /** Should not validate answer if answer is hidden by an unmet condition */
```

Comment:
Do we need comments to understand the logic?

Reply:
No, in this case it's a bit redundant; I'll remove it 👍.